csclient/csclient_test.go

```diff
@@ -304,16 +304,16 @@ func (s *suite) TestGetArchiveWithStatsDisabled(c *gc.C) {
 	s.checkCharmDownloads(c, key, 0)
 }
 
-//func (s *suite) TestStatsUpdate(c *gc.C) {
-//	key := s.checkGetArchive(c)
-//	s.checkCharmDownloads(c, key, 1)
-//	err := s.client.StatsUpdate(params.StatsUpdateRequest{
-//		CharmReference: charm.MustParseReference("~charmers/utopic/wordpress-42"),
-//		Timestamp: time.Now(),
-//		Type: params.Deploy})
-//	c.Assert(err, gc.IsNil)
-//	s.checkCharmDownloads(c, key, 2)
-//}
+func (s *suite) TestStatsUpdate(c *gc.C) {
+	key := s.checkGetArchive(c)
+	s.checkCharmDownloads(c, key, 1)
+	err := s.client.StatsUpdate(params.StatsUpdateRequest{
+		CharmReference: charm.MustParseReference("~charmers/utopic/wordpress-42"),
+		Timestamp: time.Now(),
+		Type: params.Deploy})
+	c.Assert(err, gc.IsNil)
+	s.checkCharmDownloads(c, key, 2)
```

Comment:
Add a note that this is an error and a TODO for fixing this, so this test will fail in the future.